lib/tasks/rubric.rake

```diff
@@ -54,6 +54,26 @@ namespace :db do
             assigned_groups_count:   nil
         )
       end
+      ac_with_criterion = AnnotationCategory.create(assignment: assignment,
+                                                    position: 6,
+                                                    annotation_category_name: random_words(1),
+                                                    flexible_criterion_id: assignment.flexible_criteria.first.id)
+      rand(3..12).times do |index|
```

Comment:
Yeah, I understand. I think I didn't realize this was a style issue, because I transitioned from actually using the index for the annotation texts so it's vestigial.